ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml

```diff
@@ -0,0 +1,131 @@
+import QtQuick 2.12
+import QtGraphicalEffects 1.13
+
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import utils 1.0
+import "../controls"
+import "../stores"
+
+OnboardingBasePage {
+    id: root
+
+    property string mnemonicText
+    signal exit()
+    signal seedValidated()
+
+    Item {
+        implicitWidth: 731
+        implicitHeight: 472
+        anchors.centerIn: parent
+        StatusSwitchTabBar {
+            id: switchTabBar
+            anchors.horizontalCenter: parent.horizontalCenter
+            StatusSwitchTabButton {
+                text: qsTr("12 words")
+            }
+            StatusSwitchTabButton {
+                text: qsTr("18 words")
+            }
+            StatusSwitchTabButton {
+                text: qsTr("24 words")
+            }
+            onCurrentIndexChanged: {
+                root.mnemonicText = "";
+            }
+        }
+        clip: true
+
+        GridView {
+            id: grid
+            width: parent.width
+            property var wordIndex: ["1", "5", "9", "2", "6", "10", "3", "7", "11", "4", "8", "12",
+                "13", "17", "21", "14", "18", "22", "15", "19", "23", "16", "20", "24"]
+            property var wordIndex18: ["1", "5", "9", "2", "6", "10", "3", "7", "11", "4", "8", "12",
+                "13", "", "14", "17", "15", "18", "16", ""]
+            height: (grid.count === 20 && !grid.atXBeginning) ? 144 : 244
+            anchors.left: parent.left
+            anchors.leftMargin: 12
+            anchors.top: switchTabBar.bottom
+            anchors.topMargin: ((grid.count === 20) && !grid.atXBeginning) ? 74 : 24
+            flow: GridView.FlowTopToBottom
+            cellWidth: (parent.width/4)
+            cellHeight: 72
+            interactive: false
+            model: switchTabBar.currentItem.text.substring(0,2) === "12" ? 12 :
```

Comment:
Suggestion: Could it make sense to have `States` depending on the selected tab bar and change the needed properties in transitions (like the model and maybe the construction of the SeedPhraseInput grid too)? So maybe it'd be more clear than selecting the model by subtracting a piece of the string text.

Reply:
where could States be used here, I don't quite get it? this is setting the model, thus no State changes, but just the model number, so how many delegates of the same component we will show

Reply:
> 3\. And the last thing.. I attach a video for better understanding but basically what I can see is that sometimes the word is autocompleted and sometimes not (I cannot see the pattern to give more info..). Also, sometimes it jumps to the next input and other times the user needs to click on the next one. Please, could you take a look on that?

@John-44, @alexandraB99 explained me the requirements are the following: 
```
- the autocomplete only searches the BIP39 wordlist
- the moment a user types the 3rd letter of a word, the word should be automatically completed and the focus should jump to the next word entry field.
```
But when testing it, as a user I was a bit confused because I expected the same behavior on each word input, regardless of whether it matches 3 letters or needs more. So I expected that it should be also autocompleted when there are NO MORE options regardless the number of chars that are introduced. I.e: `hammer`, when introducing `hamm` I think it should be as well autocompleted. 

Maybe there is a specific reason to only autocomplete if there is 3 characters/unique match, but like I said, as a UX perspective, for me it was confusing!